_rules/image-accessible-name-23a2a8.md

```diff
@@ -125,34 +125,34 @@ The HTML `img` element has an [accessible name][] that does not only consist of
 
 #### Failed Example 1
 
-The HTML `img` element is not marked as [decorative][] and does not have an [accessible name][].
+The HTML `img` element is not marked as [decorative][] and has an empty [accessible name][].
```

Comment:
The `img` is missing an attribute used to provide an accessible name rather than the accessible name being empty.

Reply:
As per accessible name computation, and the new note added to the definition of accessible name in that PR, this results in an empty accessible name.

Reply:
Got it thanks.